client/client-benchmark-noop-api-plugin/src/main/java/org/elasticsearch/plugin/noop/action/bulk/RestNoopBulkAction.java

```diff
@@ -85,7 +86,8 @@ public RestChannelConsumer prepareRequest(final RestRequest request, final NodeC
 
     private static class BulkRestBuilderListener extends RestBuilderListener<BulkRequest> {
         private final BulkItemResponse ITEM_RESPONSE = new BulkItemResponse(1, DocWriteRequest.OpType.UPDATE,
-            new UpdateResponse(new ShardId("mock", "", 1), "mock_type", "1", 1L, DocWriteResponse.Result.CREATED, 0L));
+            new UpdateResponse(new ShardId("mock", "", 1), "mock_type", "1", 1L,
+                DocWriteResponse.Result.CREATED, new TimeValue(1L)));
```

Comment:
We rarely call the public, long flavored ctor of `TimeValue`. I'm not actually sure why. We tend to use `timeValueMillis` or `timeValueNanos` instead. Maybe because it is more obvious the time unit. Anyway, if you want, can you change this to one of those?